itou/www/autocomplete/views.py

```diff
@@ -19,6 +19,31 @@ def sanitize(string):
     return unidecode(string.lower())
 
 
+def autocomplete_name(qs, term, department_field_name):
```

Comment:
"department" c'est pas très générique ;).
```suggestion
def generic_autocomplete_name(qs, term, extra_field_ordering):
```
Et plus qu'à changer la ligne d'en dessous en :
```
.order_by(F("best_index").asc(nulls_last=True), "name", *extra_field_ordering)
```

Pas convaincu par mon nommage (peut-être `extra_order` sinon :shrug:) mais vous voyez l'idée.